next-step-for-trello.user.js

```diff
@@ -343,6 +360,7 @@ function injectCss() {
     width: 10px;
   }
   .aj-next-step > .aj-md-hyperlink {
+  .aj-next-step > .aj-item-name > .aj-md-hyperlink {
```

Comment:
Hi @adrienjoly !
Actually, this is intended as a fix, as `.aj-next-step` has no immediate children of class `.aj-md-hyperlink`. You can see in the image I attached, or in version 1.4, that links are not underlined.

Considering the structure:

``` html
<p class="aj-next-step">
    <span class="aj-item-name">
        <span class="aj-md-hyperlink">http://mylink.com or [My link](http://mylink.com)</span>
    </span>
</p>
```

I added `.aj-item-name` in between the child selectors in order to make it work, while keeping the original selector idea. But I guess I'd just use `.aj-md-hyperlink` to improve performance a little.

Does it make sense ?